lib/skate_web/templates/layout/barebones.html.eex

```diff
@@ -17,6 +17,13 @@
         <link rel="apple-touch-icon" href="<%= static_href.(@conn, "/images/mbta-logo-t-180.png") %>" type="image/png">
         <link rel="icon" href="<%= static_href.(@conn, "/images/mbta-logo-t-favicon.png") %>" sizes="32x32" type="image/png">
         <link rel="icon" href="<%= static_href.(@conn, "/favicon.ico") %>" sizes="16x16" type="image/vnd.microsoft.icon">
+
+        <style>
+            .error-tag {
```

Comment:
Added this here because it feels like the style attached to `error_tag` ought to be available to anything in admin-land.